docs/src/ActivationCard.doc.js

```diff
@@ -0,0 +1,123 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="ActivationCard"
+    description="Activation cards are used in groups to communicate a user’s stage in a series of steps toward an overall action."
+  />
+);
+
+card(
+  <Example
+    name="Example"
+    defaultCode={`
+<Box display="flex" justifyContent="between">
```

Comment:
@norrise120 could we remove the `justifyContent="between"` and use symmetrical padding instead?
```
<Box
  display="flex"
  marginStart={-1}
  marginEnd={-1}
>
  <Box paddingX={1}>
    <ActivationCard>
  </Box>
  <Box paddingX={1}>
    <ActivationCard>
  </Box>
</Box>
```

Reply:
@norrise120 Figured it out, we need a `100%` height on the `<ActivationCard />` to make this work.

Feel free to check out the changes here: https://github.com/christianvuerings/gestalt/commit/76685a9b0851f512ba500b3ddce3424d9278f2aa

![image](https://user-images.githubusercontent.com/127199/95120362-468e6c00-0702-11eb-8773-43e546336ae8.png)

/cc @AlbertCarreras

Reply:
Updated! I also added `width="100%"` to each of the boxes holding the activation cards so that they are the same size (otherwise they look a little wonky, especially if one has a lot less content). Let me know though if there is a better way to achieve the same effect.

Before
<img width="634" alt="Screen Shot 2020-10-05 at 1 10 03 PM" src="https://user-images.githubusercontent.com/46180742/95127087-45fad300-070c-11eb-9c5b-0fe8b150767e.png">

After
<img width="849" alt="Screen Shot 2020-10-05 at 1 11 46 PM" src="https://user-images.githubusercontent.com/46180742/95127125-57dc7600-070c-11eb-844f-2723f6891f95.png">